test-data/unit/lib-stub/attr.pyi

```diff
@@ -0,0 +1,57 @@
+from typing import TypeVar, overload, Callable, Any, Type, Optional
+
+_T = TypeVar('_T')
+_C = TypeVar('_C', bound=type)
+
+@overload
+def attr(default: Optional[_T] = ...,
+         validator: Optional[Any] = ...,
+         repr: bool = ...,
+         cmp: bool = ...,
+         hash: Optional[bool] = ...,
+         init: bool = ...,
+         convert: Optional[Callable[[Any], _T]] = ...,
```

Comment:
I know other people have different opinions but I find it unfortunate that you’re adding a deprecated option to mypy. The sooner people get driven off using it, the better. Since nobody™ runs their tests with warnings activated, this might be a unique chance.

Reply:
Ok so how about I emit and error from the plugin `Using deprecated option "convert"` or something like that.  But I think we should keep the handling of it because of `--follow-imports=silent` 

Not handling `convert` means that `__init__` may not have the correct type.  

> silent follow imports normally and even “type check” them normally, but suppress any error messages. This is typically the best option for a new codebase.

Reply:
This has been added.